comps/comps-foreman-client-el10.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <!--  <meta> -->
+  <!-- Meta information will go here eventually -->
+  <!--  </meta> -->
+  <group>
+    <id>foreman-client</id>
+    <name>Foreman Client</name>
+    <description>Foreman Client Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">foreman-client-release</packagereq>
+      <packagereq type="default">foreman_ygg_migration</packagereq>
+      <packagereq type="default">foreman_ygg_worker</packagereq>
```

Comment:
@adamruzicka this fails to build on EL10:
https://download.copr.fedorainfracloud.org/results/evgeni/el10/centos-stream-10-x86_64/08343540-foreman_ygg_worker/builder-live.log.gz

Shouldn't block this PR, but something that will end up on your plate one day ;)

Reply:
Copying discussion from Matrix: after https://github.com/theforeman/foreman-packaging/pull/11493 is merged we can investigate using the proper go macros to build. Those weren't available on EL7.